effectsize_exemplar_nonparametric.Rmd

```diff
@@ -0,0 +1,46 @@
+## Exemplar: Nonparametric effect size {#effectsize_exemplar_nonparametric}
+
+For a nonparametric test that produces a Z-score, like the Mann-Whitney U test or the Wilcoxon Signed-Rank test, an effect size can be computed as:
+
+<center>$r = \left|\frac{Z}{\sqrt{N}}\right|$</center>
+
+Above, Z is the Z-score and N is the number of observations in all groups [@Rosenthal1991a, p. 19). The result, *r*, is a variance-based effect size, like Pearson *r*, not a Cohen *d*-family effect size. The *r* can be squared to estimate the percentage of variance explained, however it will not be exactly equivalent to the Pearson *r*. 
+
+```
+TODO: This needs a domain where we can argue that the nonparametric approach is an exemplar analysis, then repeat structure of exemplar 1 with it
```

Comment:
+1
Concerning the standardized effect size scenario, it could be comparing the effects of a manipulation on two dependent variables expressed in different units. For example, does my technique contribute more to improving accuracy or or to improving completion time? I did this recently. I've read Cummings and his arguments don't seem to apply to this particular use of standardized effect sizes.